nekoyume/Assets/_Scripts/UI/Module/Inventory.cs

```diff
@@ -439,26 +461,35 @@ private void UpdateDimmedInventoryItem()
             }
         }
 
+        private void SetCategoryMovable(bool movable)
+        {
+            _allowMoveTab = movable;
+        }
```

Comment:
I think this method is not necessary

Reply:
OMG Is this really my code? really? I can't understand I wrote this code... I will fix it. Thx